### ISSUE TITLE
Top nav becomes inaccessible at small viewport sizes

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#4496]
When the main navigation is in mobile view the navigation elements are missing an accessible name. When the navigation is collapsed, the text associated with each item has display: none applied to it. This attribute hides the content not just visually, but also from the accessibility tree. As a result, About Pressbooks, Administrator Network, My Books, Create Book, Clone Book, and Add Users are communicated as menu items with no name by some browser and AT combos (e.g. Firefox). 

<img width="270" height="151" alt="Image" src="https://github.com/user-attachments/assets/0c392bab-652a-4481-a8ce-ba05816da625" />

Furthermore, when the viewport is set to 320 pixels, resizing breaks the main navigation. Clone Book, Add Users and Profile buttons appear off screen requiring horizontal scrolling, and have very low contrast with the background. 

<img width="428" height="512" alt="Image" src="https://github.com/user-attachments/assets/300346ad-0967-41fb-a52e-aae331e7197b" />

**Comment by MichSpieler** [comment]
I was unable to reproduce this, but remember having been to do so a few months ago. I think this may have been addressed by Ned in a recent PR. Not sure it was in the context of this one https://github.com/pressbooks/pressbooks/pull/4456
